client/src/components/NavBar/index.tsx

```diff
@@ -6,14 +6,25 @@ import { track } from "../../analytics";
 import { EVENTS } from "../../analytics/events";
 import { ROUTES } from "./routes";
 import Icon from "../icon/icon";
-import { BetaChip, Left, LinkWrapper, MainWrapper, Nav, Right, Wrapper } from "./style";
+import {
+  BetaChip,
+  Left,
+  LinkWrapper,
+  MainWrapper,
+  Nav,
+  NavItemContainer,
+  NavSection,
+  NavSectionTitle,
+  Right,
+  Wrapper,
+} from "./style";
 import NavDivider from "./components/NavDivider";
 
 function handleMenuClick() {
   track(EVENTS.EXPLORER_MENU_BUTTON_CLICKED);
 }
 
-const CENSUS_LINK = "https://cellxgene-census.readthedocs.io/en/latest";
+const CENSUS_DOCS_LINK = "https://cellxgene-census.readthedocs.io/en/latest";
 
 interface HeaderProps {
   tosURL?: string;
```

Comment:
possible to move to a types file?

Reply:
@kaloster this is marked as resolved, but still in the index.tsx file. Was there an offline discussion that decided to keep it in this file?

Thank you!